src/stories/Library/search-result-item/search-result-item.scss

```diff
@@ -6,9 +6,9 @@
   gap: 22px 16px;
   cursor: pointer;
   text-decoration: none;
+  min-height: 184px;
 
   @include breakpoint-s {
-    height: 184px;
```

Comment:
To me it seems as if `min-height: 184px` and `height: 184px` can co-exist.  Is there any reason we do not want to force this height on specific breakpoints?

Reply:
Also: As far as I can tell from Chromatic this change shifts the current layout. With `min-height: 184px` the actual height becomes 185px but before it was forced to 184px.